routes/index.js

```diff
@@ -9,6 +9,10 @@ app.use('/contributions', require('./contributions.js'))
 app.use('/healthcheck', require('./healthCheck.js'))
 app.use('/members', require('./members.js'))
 app.use('/pullrequests', require('./pullrequests.js'))
+app.use('/contributions', require('./contributions'))
+app.use('/badges', require('./badges.js'))
+app.use('/transactions', require('./transactions.js'))
+app.use('/trade', require('./trading.js'))
```

Comment:
I haven't added this routes, this are added by other people. I think it's showing in this PR because Its old PR and I have take it over and pulled current changes on Develop branch So it might be showing here.

Reply:
yes, but it should have not shown those extra changes, Its fine though :+1:

Reply:
Yeah, They shouldn't be there. What to do know?

Reply:
Done @swarajpure